commons/plasma_sample_data_basic_query.R

```diff
@@ -0,0 +1,155 @@
+# Example script how to combine all plasma availability info with endpoint data
+# and perform some basic queries
+
+library(bigrquery)
+library(data.table)
+library(lubridate)
+library(ggplot2)
+library(bigrquery)
+
+
+# General settings
+projectid <- "fg-production-sandbox-6"
+options(gargle_oauth_cache = FALSE)
+bq_auth(scopes = "https://www.googleapis.com/auth/bigquery")
+
+# Calculate approximate event date given birth date and age at event
```

Comment:
could start getting into adding roxygen style documentation for code meant for sharing.